content/fr/blog/2021-04-27-cant-read-wont-buy.md

```diff
@@ -1,92 +1,93 @@
 ---
-title: "Can't Read, Won't Buy"
+title: "S'ils ne peuvent pas le lire, ils ne l'achèteront pas non plus"
 slug: "cant-read-wont-buy"
 date: "2021-04-27T11:24:40-04:00"
 draft: false
 image_webp: images/blog/streetblur.webp
 image: images/blog/streetblur.jpg
 author: Rebecca Bilbro
-description : "In this post we'll explore some of the drawbacks of using only English as a sales medium, check out some creative solutions, identify some areas of caution, and consider what this all means to the future of global app development."
+description : "Dans cet article, nous allons explorer certains des inconvénients de l'utilisation de l'anglais comme moyen de vente, examiner certaines solutions créatives, identifier certaines zones de précautions et réfléchir à ce que tout cela signifie pour l'avenir du développement d'applications internationales."
```

Comment:
I'm so happy that you had the metadata translated too; very nice work making sure all of the details were just right!